SimPEG/NSEM/Utils/dataUtils.py

```diff
@@ -5,25 +5,25 @@
 from scipy.constants import mu_0
 from scipy import interpolate as sciint
 
-def getAppRes(MTdata):
+def getAppRes(NSEMdata):
```

Comment:
I would suggest calling this `apparent_resistivity()` 
(starting with `get` makes looking for something in a tab completion a bit tricky in my opinion) 
Thoughts @grosenkj , @rowanc1 ?